src/pocketmine/plugin/PluginManager.php

```diff
@@ -413,7 +413,7 @@ public function isCompatibleApi(string ...$versions) : bool{
 					continue;
 				}
 
-				if($pluginNumbers[2] > $serverNumbers[2]){ //If the plugin requires bug fixes in patches, being backwards compatible
+				if($pluginNumbers[2] > $serverNumbers[2] and $pluginNumbers[1] === $serverNumbers[1]){ //If the plugin requires bug fixes in patches, being backwards compatible
```

Comment:
For clarity would probably be better to reverse the order of these checks.

Reply:
Ok.  This has been done.